pdvega/_register.py

```diff
@@ -0,0 +1,9 @@
+from ._pandas_internals import (register_dataframe_accessor,
+                                register_series_accessor)
```

Comment:
This seems to be the issue... if pandas is not a new enough version, then these names are never defined in ``_pandas_internals`` and so this leads to an import error.

I think the best way to be backward compatible would be to define a suitable version of these two functions in ``_pandas_internals`` if they aren't available in the installed pandas library.

Reply:
Ok, I think I fixed the issue! 😃 

I wrote new `register` decorators that use the `AccessorProperty` from pandas <0.23. These functions behave the same as the new extension api in pandas>=0.23. It removes the monkey patch logic and instead uses the register logic entirely.  

Is this what you were thinking?